docs/tutorials/mastering.md

```diff
@@ -85,21 +93,28 @@ Install LXD:
 sudo snap install lxd
 ```
 
-Create custom storage pool and LXD project:
+Create custom storage pool, LXD project and profile:
 
 ```console
 lxc storage create sdcore zfs size=150GiB
+```
+
+```console
 lxc project create sdcore --config features.images=true --config features.networks=true --config features.networks.zones=true --config features.profiles=true --config features.storage.buckets=true --config features.storage.volumes=true
 ```
 
+```console
+lxc profile create sdcore 
+```
+
 ```{note}
 When using LXD version lower than 5.9, the command above will produce the following error:
 > Error: Invalid project configuration key "features.networks.zones"
 
 Please upgrade your LXD to the latest version or remove unsupported config from the command.
 ```
 
-Create the configuration to initialize the LXD.
+Create the configuration to initialize the LXD:
```

Comment:
This is a bit confusing to me, we seem to be recreating a profile with `init` after we manually created it above. It seems to work, but there might be a way to simplify this.

Reply:
You're right, when you use the config file, you don't have to create all the stuff manually first (at least on my laptop it worked).